apiserver/facades/agent/uniter/uniter.go

```diff
@@ -1628,6 +1651,19 @@ func (u *UniterAPI) NetworkInfo(args params.NetworkInfoParams) (params.NetworkIn
 	if err != nil {
 		return params.NetworkInfoResults{}, err
 	}
+	defaultSpaceAddress, spaceAddressErr := machine.PrivateAddress()
```

Comment:
do this only if `args.RelationId == nil`? it's a bit wasteful to perform a DB lookup if we're not using the result